spyder/config/gui.py

```diff
@@ -91,53 +91,19 @@ def set_font(font, section='appearance', option='font'):
     FONT_CACHE[(section, option)] = font
 
 
-def get_shortcut(context, name):
-    """Get keyboard shortcut (key sequence string)"""
-    return CONF.get('shortcuts', '%s/%s' % (context, name))
-
-
-def set_shortcut(context, name, keystr):
-    """Set keyboard shortcut (key sequence string)"""
-    CONF.set('shortcuts', '%s/%s' % (context, name), keystr)
-
-
-def fixed_shortcut(keystr, parent, action):
-    """
-    DEPRECATED: This function will be removed in Spyder 4.0
-
-    Define a fixed shortcut according to a keysequence string
-    """
-    sc = QShortcut(QKeySequence(keystr), parent, action)
-    sc.setContext(Qt.WidgetWithChildrenShortcut)
-    return sc
-
-
-def config_shortcut(action, context, name, parent):
+def _config_shortcut(action, context, name, keystr, parent):
```

Comment:
I think this function should not start with `_` because it's used outside this module.

Reply:
True, but that is justa  signal to outside users to NOT use it directly :-p

Reply:
Ok, no problem then.